lib/system/linux/device.c

```diff
@@ -305,7 +308,7 @@ static void metal_uio_dev_irq_ack(struct linux_bus *lbus,
 {
 	(void)lbus;
 	(void)irq;
-	unsigned int irq_info = 1;
+	int irq_info = 1;
 	unsigned int val;
```

Comment:
why change from int to "unsigned int", please revert this change as it doesn't look like relevant to this patch series

Reply:
the opposite, changed from unsigned int to int.